.github/workflows/tests.yml

```diff
@@ -288,9 +288,9 @@ jobs:
           - "v1.11.0" # https://github.com/kubernetes/minikube/issues/8799
         kubernetes_version:
           - "v1.18.6"
-          - "v1.17.9"
-          - "v1.16.12" # v1.16.13 is broken, see https://github.com/kubernetes/minikube/issues/8840
-          - "v1.15.12"
+          #- "v1.17.9"
```

Comment:
I assume this'll be augmented by https://github.com/timberio/vector/issues/3338 ?

Reply:
Yeah! I just disabled the mid-versions for now. I'm not sure that's a safe bet, but it'll ease off a lot of load. I think #3338 is doing to augment/resolve this.